release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -140,14 +140,42 @@ xml:Text nameText = (name/*).text();
 io:println(nameText); // "DanBrown"
 ```
 
+##### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha3%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+
 #### Runtime
 
+##### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha3%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+
 #### Standard Library
 
+##### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Swan+Lake+Alpha3%22).
+
 #### Code to Cloud
 
+##### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake Alpha3 of the repositories below.
+
+- [C2C](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+project%3Aballerina-platform%2F76+)
+- [Docker](https://github.com/ballerina-platform/module-ballerina-docker/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+project%3Aballerina-platform%2F76)
+- [AWS Lambda](https://github.com/ballerina-platform/module-ballerinax-aws.lambda/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+project%3Aballerina-platform%2F76)
+- [Azure Functions](https://github.com/ballerina-platform/module-ballerinax-azure.functions/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+project%3Aballerina-platform%2F76) 
+
 #### Developer Tools
 
+To view bug fixes, see the GitHub milestone for Swan Lake Alpha3 of the repositories below.
+
+- [Language](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha3%22+label%3AType%2FBug+label%3ATeam%2FDevTools)
+- [Update Tool](https://github.com/ballerina-platform/ballerina-update-tool/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+project%3Aballerina-platform%2F32)
```

Comment:
Can we use the milestone?

Reply:
@keizer619 - Can we create the milestone in the Update Tool repo, please?

Reply:
```suggestion
```

Currently we have not fixed any bugs reported in update tools repository for this release. Improvements and Features developed there have been included in the release notes. Therefore shall we remove this for this release?

Reply:
@keizer619 - Yes, but we need to create the milestone to add it to the release note template.

Reply:
@praneesha, for the template we can just use a placeholder, right? Doesn't make sense to create an empty milestone just for this, IMO.

Reply:
Great then lets leave the entry and we need to update the link when completing the release notes

Reply:
Ack. Thanks!